psi4/driver/procrouting/dft_funcs/dict_builder.py

```diff
@@ -330,7 +331,10 @@ def build_superfunctional_from_dictionary(name, npoints, deriv, restricted):
     if "dispersion" in functionals[name]:
         d_params = functionals[name]["dispersion"]
         if "citation" not in d_params:
-            d_params["citation"] = False
```

Comment:
Small note, it looks like these are 3-space indented, not 4. Can you run this through YAPF?